recipe/meta.yaml

```diff
@@ -13,28 +13,42 @@ source:
 build:
   noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<38]
 
 requirements:
   host:
-    - python >=3.8.0
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.8.0
+    - python
     - numpy >=1.17.5
     - sympy >=1.5
 
 test:
```

Comment:
There's some tests upstream in unyt/tests, would be quite nice to implement them here